setup.py

```diff
@@ -49,6 +49,7 @@ def run_tests(self):
     "pytest-benchmark",
     "pytest-cov",
     "pytest-mock",
+    "fastdiff==0.2.0",
```

Comment:
Why is this pinned? Could use a comment for future reference.

Reply:
Comment added.